server/modules/multer.js

```diff
@@ -0,0 +1,2 @@
+const multer = require('multer');
+module.exports = { multer }
```

Comment:
I'm assuming you made this wrapper for testing? So you dont need it anymore?

Reply:
Yes, I made it for sinon stubing. I will remove it.